src/Sulu/Bundle/AdminBundle/Resources/js/containers/Datagrid/types.js

```diff
@@ -60,3 +66,7 @@ export type TreeItem = {
     data: DataItem,
     children: Array<TreeItem>,
 };
+
+export interface FieldTransformer {
+    transform(value: *): *,
```

Comment:
Are there cases where this does not return a string?

Reply:
Or to be more precise, that must be something we can use in JSX, right? So maybe use the `Node` type from react?